examples/Training/python/Advanced/Create_Image_advanced.py

```diff
@@ -24,8 +28,11 @@
 
 # Configuration
```

Comment:
This is no-longer ```#Configuration``` here but something like
" Find another image by getting first image in Dataset"